packages/components/src/call-out/types.ts

```diff
@@ -0,0 +1,33 @@
+import type { Icon } from '@wordpress/components';
+import type { ComponentProps, ReactNode } from 'react';
+
+export interface CallOutProps {
+	/**
+	 * A concise headline that clearly communicates the main message or feature being highlighted.
+	 */
+	title: string;
+	/**
+	 * A short paragraph providing supporting context or details to reinforce the title.
+	 */
+	children?: ReactNode;
+	/**
+	 * CallOut's style variant.
+	 * - default: Display the CallOut in a card.
+	 * - highlight: Gives additional prominence to the CallOut by increasing the heading size, adjusting some colors, and utilizing a primary button for the cta.
+	 * - borderless: Display the CallOut with no card, no border, no highlight.
+	 * @default 'default'
+	 */
+	variant?: 'default' | 'highlight' | 'borderless';
```

Comment:
The design spec has "card" and "highlighted" as two separate props. But it shouldn't be possible to have a non-card that is also highlighted. So I think they're better modeled as mutually exclusive.